tasks/discover-pdrs/tests/index.js

```diff
@@ -192,20 +174,14 @@ test('test pdr discovery with SFTP assuming some PDRs are new', async (t) => {
     }).promise();
 
     await validateConfig(t, event.config);
-    let output;
-    try {
-      output = await discoverPdrs(event, {});
 
-      await validateOutput(t, output);
+    const output = await discoverPdrs(event, {});
 
-      t.is(output.pdrs.length, 4);
-      const names = output.pdrs.map((p) => p.name);
-      newPdrs.forEach((pdr) => t.true(names.includes(pdr)));
-    } catch (e) {
-      if (e instanceof RemoteResourceError) {
-        t.pass('Ignoring this test. Test server seems to be down');
-      } else t.fail(e);
-    }
+    await validateOutput(t, output);
+
+    t.is(output.pdrs.length, 4);
+    const names = output.pdrs.map((p) => p.name);
+    newPdrs.forEach((pdr) => t.true(names.includes(pdr)));
```

Comment:
We need to remember this one in case we start seeing these fails pop up. Guessing they were intermittent.